index.js

```diff
@@ -0,0 +1,29 @@
+const { parse } = require('csv-parse')
+const fs = require('fs')
+const habitablePlanets = []
+
+function isHabitable (planet) { // checks if all the conditions to confirm that a planet is habitable are met
+  return planet.koi_disposition === 'CONFIRMED' &&
+    planet.koi_insol > 0.36 && planet.koi_insol < 1.11 &&

```

Comment:
1.11 is a magic number https://en.wikipedia.org/wiki/Magic_number_(programming)

Lets replace it with a named variable